src/components/VideoView.vue

```diff
@@ -44,6 +47,7 @@
 
 <script>
 import axios from 'axios'
+import AncillaryChart from '@/components/AncillaryChart.vue'
```

Comment:
I want to confirm where you want the data from the dive endpoint, AncillaryChart grabs data from dive/getCTD and AnnotationChart grabs data from the annotations endpoint.

Reply:
Are they grabbing the same data?  From what I see they're both utilizing pressure, temperature and salinity from the ancillary data. If that's the case, I would suggest to use dive/getCTD as it only returns those three

Reply:
Btw, I might remove the AnnotationChart just for now while I fix the layout of it. Then I'll go ahead and put it back in